test/suites/upgrade/upgrade-rpm-successful.robot

```diff
@@ -0,0 +1,51 @@
+*** Settings ***
+Documentation       Tests related to upgrading MicroShift on a non-ostree system
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-rpm.resource
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-process.resource
+Resource            ../../resources/ostree-data.resource
+Resource            ../../resources/ostree-health.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           rpm-based-system
+
+
+*** Variables ***
+${TARGET_REPO_URL}      ${EMPTY}
+
+
+*** Test Cases ***
+Upgrade
+    [Documentation]    Performs an upgrade from a given RPM repository URL
+    ...    and checks if it was successful
+
+    # Health of the system is implicitly checked by greenboot successful exit
+    MicroShift 413 Should Not Have Upgrade Artifacts
+    Wait Until Greenboot Health Check Exited
+
+    Uninstall MicroShift RPM Packages
+    Reboot MicroShift Host
```

Comment:
I don't think we need this here.
Original job removed RPM because our CI steps install $current version during provisioning which is 4.14, but we wanted to start from 4.13 and upgrade to 4.14 (hence the removal of initial 4.14 microshift).
Here, we're already at right starting point which is 4.13, so we should just update to new RPMs